src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -90,19 +91,19 @@ function MoneyRequestConfirmPage(props) {
 
         // Reset the money request Onyx if the ID in Onyx does not match the ID from params
         const moneyRequestId = `${iouType.current}${reportID.current}`;
-        const shouldReset = props.iou.id !== moneyRequestId;
+        const shouldReset = !isDistanceRequest && props.iou.id !== moneyRequestId;
         if (shouldReset) {
             IOU.resetMoneyRequestInfo(moneyRequestId);
         }
 
-        if (_.isEmpty(props.iou.participants) || (props.iou.amount === 0 && !props.iou.receiptPath) || shouldReset) {
+        if (_.isEmpty(props.iou.participants) || shouldReset) {
```

Comment:
Why did we delete this part of the condition? Can we add it back without breaking stuff?
`(props.iou.amount === 0 && !props.iou.receiptPath)`

Reply:
don't remember 😨